docs/concepts/workloads/controllers/deployment.md

```diff
@@ -866,13 +866,16 @@ a Pod is considered ready, see [Container Probes](/docs/concepts/workloads/pods/
 
 ### Rollback To
 
-`.spec.rollbackTo` is an optional field with the configuration the Deployment is rolling back to. Setting
-this field will trigger a rollback, and this field will be cleared every time a rollback is done.
+`.spec.rollbackTo` is an optional field with the configuration the Deployment
+should roll back to. Setting this field triggers a rollback, and this field will
+be cleared by the server after a rollback is done.
```

Comment:
I think it's worth explicitly mentioning that this should not be used declaratively (apply) due to the fact that the field is cleared by the server once the server has decided what to do with it.